spec/rspec/rails/example/system_example_group_spec.rb

```diff
@@ -58,6 +58,32 @@ module RSpec::Rails
           expect(example).to have_received(:driven_by).once
         end
       end
+
+      describe '#after' do
+        it 'sets the :extra_failure_lines metadata to an array of STDOUT lines' do
+          group = RSpec::Core::ExampleGroup.describe do
+            include SystemExampleGroup
+
+            before do
+              driven_by(:selenium)
+            end
+
+            def set_exception(*)
```

Comment:
@JonRowe I could not for the life of me figure out how to avoid "stubbing" this method. I'm sure there's something silly I'm missing, so just let me know!